spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -155,7 +154,16 @@ def send_login_request(user, password)
         return response
       else
         if response["Location"] == "/auth" # redirect to 2 step auth page
-          raise "spaceship / fastlane doesn't support 2 step enabled accounts yet. Please temporary disable 2 step verification until spaceship was updated."
+          if File.exist?(persistent_cookie_path)
+            @cookie.load(persistent_cookie_path)
+            if self.team_id.to_s.length > 0
+              return true
+            else
+              File.delete(persistent_cookie_path) # session is already invalid
+            end
+          end
```

Comment:
Interesting. Would you happen to know if that's controller by the cookie's `Max-Age` or `Expires` fields?